messages/keystore.proto

```diff
@@ -0,0 +1,12 @@
+// This function can be used to get an identifying xpub at the keypath m/4541509'/1112098098'"
+// The keypath argument has to be m/4541509'/1112098098'
+
+syntax = "proto3";
+
+message ElectrumWalletEncryptionXPubRequest {
```

Comment:
I would really prefer if the API didn't have `xpub` in it because we don't want users assuming it has something to do with xpubs when they read our API documentation that will be automatically generated from the message descriptions.

Reply:
so s/ElectrumWalletEncryptionXPubRequest/ElectrumWalletEncryptionRequest ?

Reply:
what about `ElectrumEncryptionKeyRequest`

Reply:
Ok, done renaming.